### PR TITLE
consistent thread safety check property names

### PR DESCRIPTION
**Pull request by JerryShea**
https://github.com/OpenHFT/Chronicle-Core/issues/409

### CODE REVIEW

File: src/main/java/net/openhft/chronicle/bytes/AbstractBytes.java [hunk 1]
```diff
@@ -54,7 +54,10 @@
         extends AbstractReferenceCounted
         implements Bytes<U>,
         HasUncheckedRandomDataInput {
-    protected static final boolean DISABLE_THREAD_SAFETY = Jvm.getBoolean("disable.thread.safety", false);
+    protected static final boolean DISABLE_SINGLE_THREADED_CHECK =
+            Jvm.getBoolean("disable.single.threaded.check",
+                    Jvm.getBoolean("disable.thread.safety", false));
+    protected static final boolean DISABLE_THREAD_SAFETY = DISABLE_SINGLE_THREADED_CHECK;
```
Comment (nicktindall) [inline src/main/java/net/openhft/chronicle/bytes/AbstractBytes.java]: Are we going to phase out `disable.thread.safety` in the future? should we warn if we detect people using it?

Comment (nicktindall) [reply src/main/java/net/openhft/chronicle/bytes/AbstractBytes.java]: Also I wonder if we should expose the value from AbstractCloseable and defer to that? then we could just have it detected/set in one place?

Comment (nicktindall) [reply src/main/java/net/openhft/chronicle/bytes/AbstractBytes.java]: e.g. `AbstractCloseable.areThreadSafetyChecksDisabled()`

Comment (nicktindall) [reply src/main/java/net/openhft/chronicle/bytes/AbstractBytes.java]: Or even `Jvm.areThreadSafetyChecksDisabled()` which AbstractCloseable and this both defer to?

Comment (peter-lawrey) [reply src/main/java/net/openhft/chronicle/bytes/AbstractBytes.java]: Jvm or SingleThreadedChecked methods would be best.

**Comment by JerryShea** [comment]
See also https://github.com/OpenHFT/Chronicle-Core/pull/426 and [the build](https://teamcity.chronicle.software/buildConfiguration/Chronicle_BuildAll_Release/697073?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildChangesSection=true)

**Review by nicktindall — APPROVED** [review]

**Pull request merged**